ambry-api/src/main/java/com/github/ambry/quota/QuotaMetrics.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.quota;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+
+
+/**
+ * Metrics class to capture metrics for user quota enforcement.
+ */
+public class QuotaMetrics {
```

Comment:
we probably shouldn't put QuotaMetrics here in the ambry-api subproject. It's not used by another other subproject outside of ambry-quota. I recommend putting it under ambry-quota subproject. And maybe inside AmbryQuotaManager class as an inner class.

Reply:
fixed.